cmd/package.go

```diff
@@ -192,16 +221,35 @@ func (b *PackageHelper) PackageWindows(po *ProjectOptions, cleanUp bool) error {
 	// Build syso
 	sysofile := filepath.Join(b.fs.Cwd(), basename+"-res.syso")
 
-	batfile, err := fs.LocalDir(".")
-	if err != nil {
-		return err
-	}
+	// cross-compile
+	if b.platform != runtime.GOOS {
+		folder, err := os.Getwd()
+		if err != nil {
+			return err
+		}
 
-	windresBatFile := filepath.Join(batfile.fullPath, "windres.bat")
-	windresCommand := []string{windresBatFile, sysofile, tgtRCFile}
-	err = NewProgramHelper().RunCommandArray(windresCommand)
-	if err != nil {
-		return err
+		args := []string{
+			"docker", "run", "--rm",
+			"-v", folder + ":/build",
+			"--entrypoint", "/bin/sh",
+			"wailsapp/xgo:latest",
+			"-c", "/usr/bin/x86_64-w64-mingw32-windres -o /build/" + basename + "-res.syso /build/" + basename + ".rc",
```

Comment:
I'll need to check this works. The reason `windres.bat` exists is because there was a bug around shelling in Windows/Go.

Reply:
bear in mind this is executing inside the docker container so that shouldn't matter in that case.

Reply:
yes, of course. The issue was definitely based on running Go on Windows